packages/entities/entities-certificates/vite.config.ts

```diff
@@ -16,6 +16,18 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
       entry: resolve(__dirname, './src/index.ts'),
       fileName: (format) => `${sanitizedPackageName}.${format}.js`,
     },
+    rollupOptions: {
+      external: [
+        '@kong/icons',
+        /^@kong-ui-public\/entities-shared(.*)?$/,
```

Comment:
@2eha0 I updated here to utilize a regular expression in order to not have to call out individual imports